test/run/test.utils.js

```diff
@@ -230,24 +230,28 @@ describe("lib/utils", function () {
     var args = 0;
     var expected = 1;
 
+    var defaultCanonicalPath = "/Applications/Firefox.app/Contents/MacOS/firefox-bin";
+    var devedCanonicalPath = "/Applications/Firefox Developer Edition.app/Contents/MacOS/firefox-bin";
+    var nightlyCanonicalPath = "/Applications/Firefox Nightly.app/Contents/MacOS/firefox-bin";
```

Comment:
Let's just keep the strings inline. It's a bit repetitive, but it is easier to see the expectations by just looking at the lines.